Lib/Input/ArgvInput.php

```diff
@@ -0,0 +1,361 @@
+<?php
+/**
+ * Freyja CLI Argv Input.
+ *
+ * @package Freyja\CLI\Input
+ * @copyright 2016 SqueezyWeb
+ * @since 0.1.0
+ */
+
+namespace Freyja\CLI\Input;
+
+use Freyja\CLI\Exceptions\RuntimeException;
+
+/**
+ * Represent input coming from CLI arguments.
+ *
+ * @example
```

Comment:
Seems like `@example` want the location of a file with the example.
See [documentation](http://phpdoc.org/docs/latest/references/phpdoc/tags/example.html)